### PR TITLE
NIAD-1483: Local script run against gpc 1.6.0

### PR DESCRIPTION
**Pull request by Broom94**

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -25,8 +27,59 @@ services:
...
     ports:
-      - "8110:8080"
\ No newline at end of file
+      - "8110:8080"
+  gpconnect-db:
+    image: mariadb
+    networks: 
+      - commonforgpc
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
```
Comment (michael-kainos) [inline docker/docker-compose.yml:42]: should we be storing the root password? Is this needed?

File: docker/vars.gpc-1-6-0.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+export GPC_CONSUMER_SERVER_PORT="8090"
+export GPC_CONSUMER_OVERRIDE_GPC_PROVIDER_URL="http://tkw:4854"
+export GPC_ENABLE_SDS="false"
+export GPC_CONSUMER_SDS_APIKEY="anykey"
+export GPC_CONSUMER_LOGGING_LEVEL="DEBUG"
```
Comment (michael-kainos) [inline docker/vars.gpc-1-6-0.sh:6]: should DEBUG be the default here?

**Review by michael-kainos — COMMENTED** [review]
check comments

File: docker/vars.gpc-1-6-0.sh (hunk 2 above)
Comment (Broom94) [reply docker/vars.gpc-1-6-0.sh:6]: Yeh we'll only be using it for manual testing / debugging

File: docker/docker-compose.yml (hunk 1 above)
Comment (Broom94) [reply docker/docker-compose.yml:42]: That's how NHS have set it up, nothing really on our side we can change

**Review by michael-kainos — APPROVED** [review]

**Review by dawidskainos — APPROVED** [review]

**Pull request merged**